symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -264,19 +268,26 @@ private V4MessageBlastResponse doSendBlast(List<String> streamIds, Message messa
     headers.put("sessionToken", apiClient.parameterToString(this.authSession.getSessionToken()));
     headers.put("keyManagerToken", apiClient.parameterToString(this.authSession.getKeyManagerToken()));
 
-    return apiClient.invokeAPI(
-        path,
-        "POST",
-        emptyList(),
-        null, // for 'multipart/form-data', body can be null
-        headers,
-        emptyMap(),
-        form,
-        apiClient.selectHeaderAccept("application/json"),
-        apiClient.selectHeaderContentType("multipart/form-data"),
-        new String[0],
-        typeReference
-    ).getData();
+    try {
+      return apiClient.invokeAPI(
+          path,
+          "POST",
+          emptyList(),
+          null, // for 'multipart/form-data', body can be null
+          headers,
+          emptyMap(),
+          form,
+          apiClient.selectHeaderAccept("application/json"),
+          apiClient.selectHeaderContentType("multipart/form-data"),
+          new String[0],
+          typeReference
+      ).getData();
+    } catch (ApiException e) {
+      if (e.getMessage().contains(MAXIMUM_CHARACTERS_EXCEEDED_ERROR)) {
```

Comment:
would it be enough to forward the error message and leave it as an API Exception? does the user need to handle it as a different type (i'm not sure you can recover from this)

Reply:
if so, I think we have to change nothing because the API exception will be thrown anyway? maybe we only need to log the error message?

Reply:
Another attention point is, we should ensure we do not retry when this error occurs.

Reply:
I would remove the log entirely (and the hard coded message error) this way we are not tied to a specific error message (which would be duplicated in every BDK)
It is then up to the user of the BDK to log or not this kind of error

Reply:
then we have nothing to do with this ticket :) we can easily close this PR